### ISSUE TITLE
Ionic v3 error when running ionic plugin add cordova-plugin-tesseract

### ISSUE THREAD

**Issue by karuban88** [gustavomazzoni/cordova-plugin-tesseract#25]
I got this error when running

// this is the new command now
$ ionic cordova plugin add cordova-plugin-tesseract

or

$ cordova plugin add cordova-plugin-tesseract

Error: Invalid Plugin! cordova-plugin-tesseract needs a valid package.json

**Comment by Srini-B** [comment]
Same error here too

**Comment by falkyr** [comment]
+1

**Comment by NBAMj** [comment]
You have to first [download the plugin](https://github.com/gustavomazzoni/cordova-plugin-tesseract#1-download-or-clone-this-project-copy-it-to-your-app-root-folder-and-run-ionic-command-to-add-the-plugin) at your project's root then call `ionic cordova plugin add cordova-plugin-tesseract`
It's actually mentioned in the first step in the [installation section](https://github.com/gustavomazzoni/cordova-plugin-tesseract#installation) 😄

**Comment by kirin-yuen** [comment]
@NBAMj I donwload the plugin, but when use cordova plugin add cordova-plugin-tesseract it show "Error: Invalid Plugin! cordova-plugin-tesseract needs a valid package.json", and this plugin miss package.json excatly, how can you use it?

**Comment by viksakshum** [comment]
i have the same issue. i did download outside my project and did a cp as mentioned and then went to my project root. but on running ionic cordova plugin add   it fails

**Comment by gianlazz** [comment]
Creating a file in the root of the `cordova-plugin-tesseract` folder named `package.json` and pasting in the code from the pull-request above:https://github.com/gustavomazzoni/cordova-plugin-tesseract/pull/28

Then running `ionic cordova plugin add cordova-plugin-tesseract` in the root of the project folder worked for me.

```
cli packages: (/Users/gianlazzarini/Documents/Ionic Framwork Dev/ARSpeedRead/node_modules)

    @ionic/cli-plugin-cordova       : 1.6.1
    @ionic/cli-plugin-ionic-angular : 1.4.1
    @ionic/cli-utils                : 1.7.0
    ionic (Ionic CLI)               : 3.7.0

global packages:

    Cordova CLI : 7.0.1 

local packages:

    @ionic/app-scripts : 2.0.2
    Cordova Platforms  : android 6.2.3 ios 4.4.0
    Ionic Framework    : ionic-angular 3.5.3

System:

    Node : v6.11.0
    OS   : macOS Sierra
    npm  : 3.10.10 
```

**Comment by gustavomazzoni** [comment]
Fixed by #28.
